pkg/webhook/instance_admission_test.go

```diff
@@ -441,9 +441,9 @@ func Test_triggeredPlan(t *testing.T) {
 	for _, tt := range tests {
```

Comment:
How about adding a case for setting a non-existent parameter?

Reply:
Hu? The test you [commented above ^^](https://github.com/kudobuilder/kudo/pull/1441#discussion_r400052043) is testing exactly that.

Reply:
Let's discuss in that comment.